src/core/toga/widgets/canvas.py

```diff
@@ -459,6 +493,27 @@ def color(self, value):
             self._color = parse_color(value)
 
 
+class Path(Context):
+    """A user-created :class:`Path <Path>` drawing object for a
+    path context.
```

Comment:
I'm not sure I see what Path is adding here - the implementation of `_draw()` is the only significant functionality, and the implementation is identical to Context (which is the direct descendent). How is Path any different to Context?

Reply:
I agree, did not look at Context to closely, this class does not add anything, will fix this